<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Music Player</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="loadButton">
        <property name="text">
         <string>Load CSV</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="searchInput">
        <property name="placeholderText">
         <string>Search...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="searchComboBox">
        <item>
         <property name="text">
          <string>Artist</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Title</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Genre</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="searchButton">
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="sortComboBox">
        <item>
         <property name="text">
          <string>Popularity</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Year</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Duration</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Danceability</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="songTable">
      <property name="columnCount">
       <number>6</number>
      </property>
      <column>
       <property name="text">
        <string>Artist</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Track</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Year</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Popularity</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Genre</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="addButton">
        <property name="text">
         <string>Add to Playlist</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removeButton">
        <property name="text">
         <string>Remove from Playlist</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="shuffleButton">
        <property name="text">
         <string>Shuffle Playlist</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Current Playlist</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="playlistTable">
      <property name="columnCount">
       <number>6</number>
      </property>
      <column>
       <property name="text">
        <string>Artist</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Track</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Year</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Popularity</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Genre</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>